frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.styled.jsx

```diff
@@ -13,8 +14,32 @@ export const BookmarkTypeIcon = styled(Icon)`
   opacity: 0.5;
 `;
 
-export const BookmarkLinkRoot = styled.div`
+export const BookmarkListRoot = styled.div`
   margin: ${space(2)} 0;
 `;
 
+export const BookmarkContainer = styled.div`
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  &:hover {
+    background: ${color("bg-medium")};
+
+    button {
+      opacity: 0.5;
+    }
+  }
+
+  button {
```

Comment:
I believe that it's possible to use `IconButtonWrapper` here instead of a raw `button`.

Reply:
@ranquild that's true, and it was my first path.

However, for some strange reason, I couldn't use it because then the element wouldn't render at all.

Long story, possibly to do with the hover state and so on.

I was wasting so much time and there was this obvious alternative, I went with it.